app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -0,0 +1,78 @@
+module Idv
+  module DocumentCaptureConcern
+    extend ActiveSupport::Concern
+
+    private
+
+    def save_proofing_components
+      return unless effective_user
+
+      doc_auth_vendor = DocAuthRouter.doc_auth_vendor(
+        discriminator: document_capture_session_uuid,
+        analytics: analytics,
+      )
+
+      component_attributes = {
+        document_check: doc_auth_vendor,
+        document_type: 'state_id',
+      }
+      ProofingComponent.create_or_find_by(user: effective_user).update(component_attributes)
+    end
+
+    def track_document_state(state)
```

Comment:
Minor nit. Can this be `track_document_issuing_state`? I feel like `document_state` is generic-sounding enough that I don't necessarily link it automatically to a US state.

Reply:
@matthinz  - fair enough ... done!